web/crux-ui/src/components/nodes/dyo-node-setup.tsx

```diff
@@ -33,6 +33,7 @@ const DyoNodeSetup = (props: DyoNodeSetupProps) => {
     node.install ? expiresIn(new Date(node.install.expireAt)) : null,
     () => onNodeInstallChanged(null),
   )
+  const [rootPath, setRootPath] = useState<string>('')
```

Comment:
It could take the default path `/srv....` - default value.

Reply:
It's different on linux and mac 😅 I could also just use null.

Reply:
That is why it is important to be explicit about it, even a select of reasonable defaults could guide the user a bit.